### ISSUE TITLE
Can i solve questions here too and create pr's?

### ISSUE THREAD

**Issue by Vivekyadavgithub** [TarunSingh56/DSA-CRACKER#2]
Wanna do this sheet together?? which topic u on now

**Comment by Vivekyadavgithub** [comment]
i can do like questionname_myname.cpp if thats ok with you

**Comment by TarunSingh56** [comment]
Yes, sure you can make PR's and add your solution. However, mention all the possible approaches from brute to optimal and the time and space complexities clearly. Also, explain your approach in the comments.
Don't forget to update the readme file as well.

**Comment by Vivekyadavgithub** [comment]
So do u follow any schedule like five question per day!!! or anything like that and should i create seperate folder with my name having two solution of same question can be messy in same folder

**Comment by Vivekyadavgithub** [comment]
and i am solving trees and we only have to write function in some question so do i have to write all of it or just function here??

**Comment by thetarunsingh** [comment]
Hi Vivek, sorry for the late reply.
No need to create a separate folder. Just name your file as questionname.cpp as usual.  
To avoid any duplication, add only those solutions which are not present in the repo.

Don't worry all the contributors will be mentioned clearly in the readme file.

**Comment by buckk-UKM** [comment]
hey buddy how you made that readme file so much better. Have you made all the stuff manually or used something else. Please tell.

**Comment by thetarunsingh** [comment]
Hi, thank you, 
So you write a good Readme file with markdown syntax which GitHub provides for a better user experience.
You can find here [Markdown Cheatsheet](https://github.com/tchapi/markdown-cheatsheet/blob/master/README.md) rules.

Also, you can look into my readme file, which markdown syntax I used.
I hope, this helps and yes I did it all.
Good Luck :rocket:

**Comment by ggautamgoyalg** [comment]
hi
is this still open

**Comment by ggautamgoyalg** [comment]
We can add some sections like game theory and many more

**Comment by tarunsinghdev** [comment]
Sure, we can do that. We can create a folder name `Extras` and inside that, we can add respective folders for the new topics that are not included in [this](https://drive.google.com/file/d/1FMdN_OCfOI0iAeDlqswCiC2DZzD4nPsb/view) sheet.
